### PR TITLE
Add com.nickgirga.webready

### PR DESCRIPTION
**Pull request by nickgirga**
### Please confirm your submission meets all the criteria

Please describe your application briefly below.

WebReady is a tool to make WEBP animations from video files in a matter of seconds. You can simply right-click a video, open it with WebReady, and press ENTER. By default, it creates WEBP animated images that are configured perfectly for use as live wallpapers in Gaming Mode on the Steam Deck. This is its primary purpose.

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*

`--filesystem=home` is used to easily access chosen video files and the path to the output file. This also simplifies the "Open With" option for automatically choosing the file that the user opened WebReady with.
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.

- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: com.nickgirga.webready.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  - --filesystem=home
+
+modules:
+  - name: webready
+    buildsystem: simple
+    build-commands:
+      - install -D webready /app/bin/webready
+      - install -D webready.glade /app/share/webready/webready.glade
+      - install -D res/icons/scalable.svg /app/share/icons/hicolor/scalable/apps/com.nickgirga.webready.svg
+      - install -D res/icons/128x.png /app/share/icons/hicolor/128x128/apps/com.nickgirga.webready.png
```
Comment (bbhtt) [inline com.nickgirga.webready.yml:23]:
```suggestion
```

svg is enough

File: com.nickgirga.webready.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  - name: libhandy
+    buildsystem: meson
+    config-opts:
+      - -Dglade_catalog=disabled
+      - -Dtests=false
+      - -Dexamples=false
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/libhandy
+        tag: 1.0.3
```
Comment (bbhtt) [inline com.nickgirga.webready.yml:49]: Do you need 1.0.3? Runtime has 1.8.3

**Comment by hfiguiere** [comment]
This is missing:

> Please describe your application briefly below.

File: com.nickgirga.webready.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+sdk: org.gnome.Sdk
+command: webready
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # access to home files; for locating WADs and custom installed source ports
```
Comment (hfiguiere) [inline com.nickgirga.webready.yml:13]: What's the relation with this comment and what the app seems to do?

File: com.nickgirga.webready.yml (hunk 1 above)
Comment (nickgirga) [reply com.nickgirga.webready.yml:23]: I was using the 128x PNG as a resource for the about page. Just updated it to create a Pixbuf from the SVG. Will create a new release tag and update the flatpak manifest and stop including the PNG.

File: com.nickgirga.webready.yml (hunk 2 above)
Comment (nickgirga) [reply com.nickgirga.webready.yml:49]: Oh, great. I didn't know that. I will remove libhandy as a module.

File: com.nickgirga.webready.yml (hunk 3 above)
Comment (nickgirga) [reply com.nickgirga.webready.yml:13]: Sorry. I used an old manifest of mine for another app as a template. Forgot to update the comment. The permission is needed for this app. Will update it.

**Comment by nickgirga** [comment]
> This is missing:
> 
> > Please describe your application briefly below.

Sorry. I thought filling out the checkboxes WAS describing the app. I'll update the MR/PR with an accurate text description of what my app does.

**Comment by nickgirga** [comment]
The suggested changes have been made and I verified that my app still builds and runs correctly. I also verified that the manifest and the repo do not throw any warnings or errors with the linter.

File: com.nickgirga.webready.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+command: webready
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # access to home files; for easy access to videos and "Open With"
+  - --filesystem=home
```
Comment (bbhtt) [inline com.nickgirga.webready.yml:14]: Seems to use the portal file. Doesn't seem to be required.

Comment (nickgirga) [reply com.nickgirga.webready.yml:14]: The file chooser works okay without that permission, but the output file (which should end up right next to the source video file) will end up in the "/run/user/..." directories instead, which is not desirable behavior.

Comment (bbhtt) [reply com.nickgirga.webready.yml:14]: Ah right forgot about that

**Pull request closed without merging**